### PR TITLE
Update rate limit section in limits.md with new graphql rate limit

### PR DESCRIPTION
**Pull request by harukatab**
## Description

What does the pull request do? If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Ticket

- Update the rate limit section to add new documentation on rate limiting graphql APIs. 
- Fixed some typos
- Reordered parts within the rate limit section
- draft: https://www.notion.so/wandbai/Public-doc-graphql-rate-limits-12de2f5c7ef3804cb58ef6f675e0dd6e?pvs=4

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [ ] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [ ] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [ ] I merged the latest changes from `main` into my feature branch before submitting this PR.

### CODE REVIEW

File: docs/guides/track/limits.md [hunk 1]
```diff
@@ -152,30 +149,28 @@
 
 ### Run count
 
-For faster loading times, keep the total number of runs in a single project under 10,000.  Large run counts can slow down project workspaces and runs table operations, especially when grouping is enabled or runs have a large count of distinct metrics.  
-
-If you find that you or your team are frequently accessing the same set of runs (for example, recent runs), consider [bulk moving *other* runs](/guides/app/features/runs-table#move-runs-between-projects) to a new project used as an archive, leaving a smaller set of runs in your working project.  
+For faster loading times, keep the total number of runs in a single project under 10,000. Large run counts can slow down project workspaces and runs table operations, especially when grouping is enabled or runs have a large count of distinct metrics.
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md:152]: didn't change content

File: docs/guides/track/limits.md [hunk 2]
```diff
@@ -69,17 +67,16 @@ with f as open("large_file.json", "r"):
 Wide values can affect the plot load times for all metrics in the run, not just the metric with the wide values.
 
 :::info
-Data is saved and tracked even if you log values wider than the recommended amount. However, your plots may load more slowly. 
+Data is saved and tracked even if you log values wider than the recommended amount. However, your plots may load more slowly.
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md:70]: didn't change content, just fixed an extra space

File: docs/guides/track/limits.md [hunk 3]
```diff
@@ -152,30 +149,28 @@
...
 
-
 ### File count
 
 Keep the total number of files uploaded for a single run under 1,000. You can use W&B Artifacts when you need to log a large number of files. Exceeding 1,000 files in a single run can slow down your run pages.
 
 ## Python script performance
 
-There are a few ways that your performance of your python script is reduced:
+There are a few ways that the performance of your python script is reduced:
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md:170]: Didn't change the context, just changed your to the

File: docs/guides/track/limits.md [hunk 4]
```diff
@@ -186,53 +181,59 @@
...
 
-The proceeding table describes rate limit HTTP headers:
+### Rate limits on metric logging API
 
-| Header name | Description |
-| ----- | ----- |
-| RateLimit-Limit | The amount of quota available per time window, scaled in the range of 0 to 1000 |
-| RateLimit-Remaining | The amount of quota in the current rate limit window, scaled in the range of 0 and 1000 | 
-| RateLimit-Reset | The number of seconds until the current quota resets |
+The `wandb.log` calls in your script utilize a metrics logging API to log your training data to W&B. This API is engaged through either online or [offline syncing](../../ref/cli/wandb-sync.md). In either case, it imposes a rate limit quota limit in a rolling time window. This includes limits on total request size and request rate, where latter refers to the number of requests in a time duration.
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md:202]: Didn't change content, just changed the ordering

File: docs/guides/track/limits.md [hunk 5]
```diff
@@ -186,53 +181,59 @@
...
+
 ```python
 if epoch % 5 == 0:  # Log metrics every 5 epochs
     wandb.log({"acc": accuracy, "loss": loss})
-```  
-- Manual data syncing: Your run data is stored locally if you are rate limited. You can manually sync your data with the command `wandb sync <run-file-path>`. For more details, see the [`wandb sync`](../../ref/cli/wandb-sync.md) reference.
+```
 
-## Browser considerations
+- Manual data syncing: Your run data is stored locally if you are rate limited. You can manually sync your data with the command `wandb sync <run-file-path>`. For more details, see the [`wandb sync`](../../ref/cli/wandb-sync.md) reference.
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md]: updated

File: docs/guides/track/limits.md [hunk 6]
```diff
@@ -186,53 +181,59 @@
...
 
-The W&B app can be memory-intensive and performs best in Chrome.  Depending on your computer's memory, having W&B active in 3+ tabs at once can cause performance to degrade; if you encounter unexpectedly slow performance, consider closing other tabs or applications.
+### Rate limits on GraphQL API
 
+The W&B Models UI and SDK’s [public API](https://docs.wandb.ai/ref/python/public-api/api) make GraphQL requests to the server for data querying and mutations. For all GraphQL requests in SaaS Cloud, W&B applies rate limits per IP address for unauthorized requests and per user for authorized requests. These limits follow a fixed rate (requests per second) within a fixed time window, where your pricing plan determines the default limits. For relevant SDK requests that specify a project path (e.g., reports, runs, artifacts), W&B applies rate limits per project, measured by database query time.
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md]: updated

File: docs/guides/track/limits.md [hunk 7]
```diff
@@ -186,53 +181,59 @@
...
 
+If you are fetching a large volume of data using the W&B Models SDK's [public API](https://docs.wandb.ai/ref/python/public-api/api), consider waiting at least one second between requests. If you receive a `429` status code or see `RateLimit-Remaining=0` in the response headers, wait for the number of seconds specified in `RateLimit-Reset` before retrying.
 
+## Browser considerations
 
+The W&B app can be memory-intensive and performs best in Chrome. Depending on your computer's memory, having W&B active in 3+ tabs at once can cause performance to degrade; if you encounter unexpectedly slow performance, consider closing other tabs or applications.
 
+## Reporting performance issues to W&B
 
+W&B takes performance seriously and investigates every report of lag. To expedite investigation, when reporting slow loading times consider invoking our built-in performance logger that captures key metrics and performance events. Append &PERF_LOGGING to your URL, and share the output of your console.
```
[the remark this answers is not in the document]
Comment (harukatab) [reply docs/guides/track/limits.md]: updated

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**